package.json

```diff
@@ -42,7 +42,7 @@
   },
   "homepage": "https://github.com/hmrc/hmrc-frontend#readme",
   "dependencies": {
-    "govuk-frontend": "^3.12.0"
+    "govuk-frontend": "^3.13.0"
```

Comment:
@matthewmascord do you think we should remove the caret? Probably not really risky